code/_core/obj/structure/totem.dm

```diff
@@ -226,3 +230,135 @@
 			continue
 		L.Move(get_step(L.loc,get_dir(L,src)))
 		CREATE(/obj/effect/temp/electricity,L.loc)
+
+/obj/structure/totem/frost_spray
+	name = "totem of frost spray"
+	desc = "The weather outside is frightful, but the fire is so delightful."
+	desc_extended = "A totem that will fire frost spray at the caster's enemies."
+	icon_state = "frost"
+
+/obj/structure/totem/frost_spray/totemic_effect() //will need testing and help to balance this
+	var/turf/T = get_turf(src)
+	var/list/chooseEnemies = list()
+	for(var/mob/living/L in viewers(4,T))
+		if(L.dead)
+			continue
+		if(L.loyalty_tag == affecting_faction)
+			continue
+		if(!istype(L.health))
+			continue
+		chooseEnemies += L
+	if(chooseEnemies == list())
+		return
+	for(var/i in 1 to leveled_effect)
+		shoot_projectile(src, pick(chooseEnemies), null, null, /obj/projectile/magic/frost, /damagetype/ranged/magic/frost, 16, 16, 0, TILE_SIZE*0.5, 1, "#FFFFFF", 0, 0, 1, affecting_faction, affecting_faction)
+
+/obj/structure/totem/flame_spray
+	name = "totem of flame spray"
+	desc = "The weather outside is frightful, but the fire is so delightful."
+	desc_extended = "A totem that will fire flame spray at the caster's enemies."
+	icon_state = "flame"
+
+/obj/structure/totem/flame_spray/totemic_effect() //will need testing and help to balance this
+	var/turf/T = get_turf(src)
+	var/list/chooseEnemies = list()
+	for(var/mob/living/L in viewers(4,T))
+		if(L.dead)
+			continue
+		if(L.loyalty_tag == affecting_faction)
+			continue
+		if(!istype(L.health))
+			continue
+		chooseEnemies += L
+	if(chooseEnemies == list())
+		return
+	for(var/i in 1 to leveled_effect)
+		shoot_projectile(src, pick(chooseEnemies), null, null, /obj/projectile/magic/lesser_fire, /damagetype/ranged/magic/flame, 16, 16, 0, TILE_SIZE*0.5, 1, "#FFFFFF", 0, 0, 1, affecting_faction, affecting_faction)
+
+/obj/structure/totem/shock_spray
+	name = "totem of shock spray"
+	desc = "Listen to be baby, you've got to understand, lightning striking again!"
+	desc_extended = "A totem that will fire shock spray at the caster's enemies."
+	icon_state = "shock"
+
+/obj/structure/totem/shock_spray/totemic_effect() //will need testing and help to balance this
+	var/turf/T = get_turf(src)
+	var/list/chooseEnemies = list()
+	for(var/mob/living/L in viewers(4,T))
+		if(L.dead)
+			continue
+		if(L.loyalty_tag == affecting_faction)
+			continue
+		if(!istype(L.health))
+			continue
+		chooseEnemies += L
+	if(chooseEnemies == list())
```

Comment:
You should use ``if(!length(chooseEnemies))`` instead. Lists don't work this way.